### DISCUSSION TITLE
Does mikro-orm support oracle database?

### DISCUSSION THREAD

**Discussion by miesgre** [mikro-orm/mikro-orm#5742] [Q&A]
I am developing a new application and I need to connect to an Oracle database.
Does mikro-orm support oracle database?
If not, is it on the road map?

**Comment by B4nan** [comment]
Not supported at this moment, there is some WIP in #825 but it will take some time to finish (and I have zero experience with oracle, so I am not the best person to do it). 

So on a roadmap, yes, but with low priority, most likely won't be finished during this year.
